solr/core/src/java/org/apache/solr/handler/component/DebugComponent.java

```diff
@@ -181,16 +183,28 @@ public void modifyRequest(ResponseBuilder rb, SearchComponent who, ShardRequest
     }
   }
 
+  @VisibleForTesting
+  static String getDistributedStageName(int stage) {
```

Comment:
minor/subjective: making this non-static so that custom debug components could override to use an alternative (more meaningful) stage name
```suggestion
  protected String getDistributedStageName(int stage) {
```

Reply:
#2594 opened with the changes in this PR plus making the new method protected with matching test changes.